### ISSUE TITLE
Cursor gets stuck in browse mode

### ISSUE THREAD

**Issue by Danstiv** [nvaccess/nvda#16608]
### Steps to reproduce:
1. Visit [that page](https://stackoverflow.com/questions/58491123/sqlalchemy-accessing-attribute-from-bi-directional-association-proxy).
2. Navigate to first level heading.
3. Read up to line "`account = db.relationship("Account", backref=backref('user_accounts"`
4. Try to read some previous line by pressing up arrow repeatedly.
### Actual behavior:
NVDA reads the empty line several dozen times, and only after that starts reading the previous lines.

### Expected behavior:
NVDA reads the empty line one time only, and then reads the previous lines.

#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
alpha-32125,50da9987 (2024.3.0.32125)
#### Windows version:
Windows 11 23H2 (AMD64) build 22631.3593
#### Name and version of other software in use when reproducing the issue:
Google Chrome '122.0.6261.112'
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Seen this problem in 2022.x
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by Adriani90** [comment]
Is automatically move focus to focusable elements enabled in NVDA browse mode settings?

**Comment by Danstiv** [comment]
@Adriani90 
"Automatically set system focus to focusable elements" is disabled, but when enabled, the behavior remains the same.
Also, enabling this setting noticeably degrades the experience of working with interactive documents, so it is not a solution in any case.

**Comment by Adriani90** [comment]
You have to set the number of characters per line in browse mode to 121 or 122, anything below that will make this issue happen. I don't know why this happens.
Anyway, closing as duplicate of #2812.
